src/index.js

```diff
@@ -44,15 +44,17 @@ const handler = (req, res) => {
     if (err) {
       // eslint-disable-next-line no-console
       console.error(err);
-      res.writeHead(err.status || err.statusCode || 500);
-      res.end(err.message);
+      res.statusCode = err.status || err.statusCode || 500;
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify({ error: err.message }));
```

Comment:
For GraphQL compatibility, this should be
```suggestion
      res.end(JSON.stringify({ errors: [{message: err.message}] }));
```